### PR TITLE
Set LIBGUESTFS_BACKEND=direct for kvm guests

### PR DESCRIPTION
**Pull request by chloerh**
Since there's an unresolved issue for v2v conversion.

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

File: v2v/tests/src/convert_vm_to_ovirt.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def run(test, params, env):
         v2v_params.update({'output_format': 'qcow2'})
 
     # Set libguestfs environment variable
-    if hypervisor == 'xen':
+    if hypervisor in ('xen', 'kvm'):
```
Comment (balamuruhans) [inline v2v/tests/src/convert_vm_to_ovirt.py]: do we see unresolved issue in libvirtd with respect to v2v ?

Comment (chloerh) [inline v2v/tests/src/convert_vm_to_ovirt.py]: This issue was found under circumstances of guest image located in NFS storage which v2v had no writing permission. Maybe not affect libvirtd?

Comment (balamuruhans) [inline v2v/tests/src/convert_vm_to_ovirt.py]: you have tried permitting selinux for libvirt to use nfs,
```
# getsebool virt_use_nfs 
virt_use_nfs --> off
# setsebool virt_use_nfs on
# getsebool virt_use_nfs 
virt_use_nfs --> on
```

**Comment by will-Do** [comment]
A detail desc about why doing this is necessary

**Comment by chloerh** [comment]
This is a workaround for an existing problem which not only affects test result, will be fixed in the future.

**Comment by chloerh** [comment]
Recent test result (kvm only):


kvm.linux.5_11.i386.raw_f.NFS | 11 分 | Passed
kvm.linux.6_9.i386.raw_f.NFS | 14 分 | Passed
kvm.linux.6_9.x86_64.qcow2_f.NFS | 14 分 | Passed
kvm.linux.7_4.x86_64.qcow2_f.NFS | 15 分 | Passed
kvm.windows.win10.i386.qcow2_f.NFS | 16 分 | Passed
kvm.windows.win2008.i386.raw_f.NFS | 16 分 | Passed
kvm.windows.win2016.x86_64.qcow2_f.NFS | 21 分 | Fixed
kvm.windows.win7.i386.qcow2_f.NFS | 22 分 | Skipped
kvm.windows.win8.x86_64.raw_f.NFS | 18 分 | Passed
kvm.windows.win8_1.i386.qcow2_f.NFS | 18 分 | Passed

File: v2v/tests/src/convert_vm_to_ovirt.py (hunk 1 above)
Comment (kylazhang) [reply v2v/tests/src/convert_vm_to_ovirt.py]: @chloerh do you try the method above?

Comment (chloerh) [reply v2v/tests/src/convert_vm_to_ovirt.py]: I'll ask @weikunzz to try this

Comment (weikunzz) [reply v2v/tests/src/convert_vm_to_ovirt.py]: @balamuruhans Hi,I have tried it according to what you said, but it does not work.Thanks

Comment (balamuruhans) [reply v2v/tests/src/convert_vm_to_ovirt.py]: @weikunzz, Thank you. I just want to make sure selinux doesn't hinders your tests. cool then, let us merge this patch for the fix.

**Comment by balamuruhans** [comment]
Thanks all.

**Pull request merged**